source/typescript.txt

```diff
@@ -0,0 +1,29 @@
+.. _typescript-center:
+
+==========
+TypeScript
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+Introduction
+------------
+
+Welcome to the documentation site for official MongoDB TypeScript support.
+Since TypeScript builds on JavaScript, you can use the
+`{+node-driver-long+} <https://www.mongodb.com/docs/drivers/node/current/>`__
+to create TypeScript applications that communicate with MongoDB.
+
+Versions 4.0 and later of the {+node-driver-short+} support TypeScript. You must
+install that version of the driver or a more recent one to use TypeScript.
```

Comment:
Suggestion:

I think "support Typescript" could be more precise -- it's possible for any Javascript to "support" it by using the "any" type. Instead, it could be better to mention that the driver supports using Typescript types with it in your app.

I think the two sentences could also be combined into one.


```suggestion
You must use v4.0 or later of the driver to use Typescript types in your app. 
```